src/app/components/AprendaComOGrace.tsx

```diff
@@ -0,0 +1,99 @@
+import { ReactElement } from "react";
+import { FiArrowRight } from "react-icons/fi";
+
+interface MediaCardProps {
+    src: string
+    alt: string
+    title: string
+    sneak_peek_text: string
+    link: string
+}
+
+function MediaCard(props: MediaCardProps) {
+    return (
+        <div className="flex items-center text-sm bg-vermelhoGrace rounded-xl">
+            <img src={props.src} alt={props.alt} />
+            <div className="flex flex-col justify-items text-brancoGrace h-full w-full">
+                <h1 className="font-bold">{props.title}</h1>
+                <h2 className="w-5/6">{props.sneak_peek_text + "..."}</h2>
+                <a
+                    href={props.link}
+                    className="flex items-center self-end mr-8 text-brancoGrace font-semibold"
+                >
+                    Ler tudo <FiArrowRight />
+                </a>
+            </div>
+        </div>
+    )
+}
+
+interface TopicProps {
+    title: string
+    desc: string
+    link_text: string
+    link: string
+    embed: ReactElement
+}
+
+function Topic(props: TopicProps) {
+    return (
+        <div className="mb-12">
+            <h2 className="font-bold text-2xl text-cinzaGrace mb-8">{props.title}</h2>
+            <p className="text-lg mb-8">{props.desc}</p>
+            {/* TODO: Trocar pelo componente*/}
+            <a
+                href={props.link}
+                className="font-semibold text-vermelhoGrace flex items-center mb-12">
+                {props.link_text + " "} <FiArrowRight />
+            </a>
+            {props.embed}
+        </div>
+    )
+}
+
+export default function AprendaComOGrace() {
+    return (
+        <div className="grid grid-cols-2 gap-4">
```

Comment:
:hammer: **Ajuste** | 📱 **Responsividade**

Sugiro que, quando a tela for md ou menor, deixar a grace embaixo do conteúdo ao invés do lado `(grid-cols-1)`

![Captura de tela 2024-05-28 - 11 49 55](https://github.com/petsi-each/Grace-Website/assets/62656745/8d66687a-6365-436a-b862-d4fa71e2844f)

![Captura de tela 2024-05-28 - 11 50 56](https://github.com/petsi-each/Grace-Website/assets/62656745/b5f9fe1f-6e89-41c1-94b2-53485efac2b5)

Reply:
Tava ficando estranho, meia página só da foto dela. Resolvi esconder a imagem mesmo quando a tela é suficientemente pequena. 


https://github.com/petsi-each/Grace-Website/assets/81436626/9016ac14-0bc6-4d64-8e4a-dc265a1bc73a

Reply:
Sim sim, nesse caso, a gente pode quebrar o grid!

Mesmo escondendo a tela dela, o seu conteúdo tá no cantinho, isso pq nosso grid ainda tem 2 colunas (mas uma delas tem uma imagem escondida)

Então a gente pode fazer `grid-cols-1 lg: grid-cols-2` por exemplo

Nesse caso, se a tela é pequena, vamos ter 1 coluna só (aí o conteúdo fica em cima e a foto embaixo, por ex) e, se for maior, a gnt coloca um do lado do outro